Capstone/View Controllers/AccountViewController.h

```diff
@@ -0,0 +1,19 @@
+//
+//  AccountViewController.h
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/3/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <FBSDKLoginKit/FBSDKLoginKit.h>
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AccountViewController : UIViewController<FBSDKLoginButtonDelegate>
+@property (weak, nonatomic) IBOutlet UIView *fbLogoutView;
```

Comment:
any reason why this is public instead of private within the .m file?

Reply:
I assumed that I might want to reuse it later on in the profile view controller as the button updates to "Logout" after signing in.